frontend/src/Components/Addtask.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import styles from "@/styles/Addtask.module.css"
+import { RxCross2 } from "react-icons/rx";
+import { FaTag } from "react-icons/fa6";
+import { IoMdPersonAdd } from "react-icons/io";
+import MemberCard from "./MemberCard";
+
+const Addtask = () => {
+    const [subject, setSubject] = useState("");
+    const [details, setDetails] = useState("");
+    const [label, setLabel] = useState("Add Labels"); // Default label
+    const [selectedMember, setSelectedMember] = useState<{
+        name: string;
+        pic: string;
+    }>({ name: "Add member", pic: "https://picsum.photos/200" }); // Default selected member
+
+    const members = [
+        { name: "birla", pic: "https://picsum.photos/100" },
+        { name: "pranjal", pic: "https://picsum.photos/400" },
+        { name: "12abc", pic: "https://picsum.photos/600" },
+        { name: "xae12", pic: "https://picsum.photos/300" },
+    ];
+
+    const handleSubjectChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setSubject(e.target.value);
+    };
+
+    const handleDetailsChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+        setDetails(e.target.value);
+    };
+
+    const handleLabelClick = (clickedLabel: string) => {
+        setLabel(clickedLabel === label ? "Add Labels" : clickedLabel); // Toggle label
+    };
+
+    const handleMemberClick = (name: string, pic: string) => {
+        setSelectedMember({ name, pic }); // Update selected member
+    };
+
+    const handleResetForm = () => {
+        setSubject("");
+        setDetails("");
+        setLabel("Add Labels");
+        setSelectedMember({ name: "Add member", pic: "https://picsum.photos/200" });
+    };
+
+    const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+
+        console.log("Task created:", { subject, details, label, selectedMember });
+        handleResetForm();
+    };
+    return (
+        <>
+            <div className={styles.addTask}>
```

Comment:
why are you still using the dynamic classes ?

Reply:
Actually this component is not created by me , and if i recreate this then i have to do some changes